src/tokens/index.js

```diff
@@ -0,0 +1,4 @@
+import colors from './colors';
+import fonts from './fonts';
+
+export { colors, fonts };
```

Comment:
I think this should not be plural. When we consume it, it would make more sense if it wasn't plural

eg. 
```color.green.300```
```font.size.m```

Reply:
It's a collection of properties stored in object literal. So it should be plural. Because we are saying in the collection of colors choose green of 300

Reply:
Android follows a similar pattern https://developer.android.com/reference/android/R.color

So does treebo :P https://github.com/treebohotels/leaf-ui/blob/master/src/theme/theme.web.js

Reply:
Lol we made a mistake at Treebo hence recommended plural 😄